QUARKUS-3866.md

```diff
@@ -0,0 +1,43 @@
+# QUARKUS-3866 Dev support and productization for quarkus-security-jpa
+- JIRA link: https://issues.redhat.com/browse/QUARKUS-3866
+- Quarkus Guides: 
+  - https://quarkus.io/guides/security-getting-started-tutorial
+  - https://quarkus.io/guides/security-jpa
+
+## Scope of the testing
+- Exploratory testing of the feature
+- Basic test development in case of missing test coverage
```

Comment:
Please either remove this line or describe scenario which will constitute this "Basic test".

Side note: Ideally, test plan should contain enough info, that anyone from our team can implement the necessary coverage using only it and the guides, and we can use it as a SSoT to check the required coverage later. Conditionals like this "in case" mean, that we will have to check upstream coverage as well, which means duplicated effort (since you did it already).

Reply:
I changed it to add test coverage for different DBs and try custom password providers